app/Policies/NewsPolicy.php

```diff
@@ -37,27 +39,43 @@ public function view(?User $user, News $news): bool
     public function create(User $user): bool
     {
         return $user->hasAnyRole([
+            Role::ADMINISTRATOR,
+            Role::DEVELOPER_STAFF,
+            Role::DEVELOPER,
+            Role::EVENT_MANAGER,
+            Role::MODERATOR,
             Role::NEWS_MANAGER,
+            Role::TEAM_ACCOUNT,
```

Comment:
Do we really want to allow any developer to post news items? I think EVENT_MANAGER and NEWS_MANAGER are probably all that's needed. If an administrator, moderator, or team account wants to create news postings, they can be given the NEWS_MANAGER role.

I could maybe see allowing MODERATOR to update.

Reply:
I don't think we want to allow any developer to post news items - this is mostly to retain current functionality until the legacy page is deleted.